config.json

```diff
@@ -23,11 +23,10 @@
         ]
     },
     "redis": {
-      "host": "127.0.0.1",
-      "port": 6380,
-      "db": 0,
-      "user": "",
-      "password": "vgeuwrKZLYwNyZXNc7vTFbjJ4vQnMGrs"
+        "host": "127.0.0.1",
+        "port": 6380,
+        "db": 0,
+        "password": null
```

Comment:
where did the user field go?

why is the password null instead of an empty string?